.github/workflows/validate.yml

```diff
@@ -52,6 +52,6 @@ jobs:
       - name: Pytest coverage comment
         uses: MishaKav/pytest-coverage-comment@v1.1.51
         with:
-          github-token: ${{ secrets.ACTION_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Switch to using the builtin token github provides for actions instead of the repo specific secret so this can run in forks.

Reply:
I know there was a reason I had to switch to a repo-specific token, but the reason escapes me. I'm fine to try the regular token for now -- it's easy enough to revert this if needed.

Reply:
Here's the reason. The normal token isn't able to comment on PRs:

![image](https://github.com/user-attachments/assets/b09ad94f-2d66-4af6-82ac-37bd96e923b9)